.github/workflows/test.e2e.yml

```diff
@@ -28,4 +28,9 @@ jobs:
         run: ./scripts/build.sh -r
       - name: Run e2e tests
         shell: bash
-        run: scripts/tests.e2e.sh ./build/avalanchego
+        run: ARCHIVE_NETWORK_DIR_ON_TEARDOWN=1 scripts/tests.e2e.sh ./build/avalanchego
+      - name: Upload archive of testnet fixture dir
+        uses: actions/upload-artifact@v3
+        with:
+          name: testnet-data
+          path: testnet.tar.xz
```

Comment:
can we define a retention policy or maybe could we add a comment on what that is?

Reply:
how much data is this and what happens if we hit the limit?